FIRBoundaries.dat

```diff
@@ -1188,10 +1194,12 @@ CZQX|1|1|8|45.000000|-63.000000|65.000000|-30.000000|55.000000|-46.500000
 48.493611|-8.00000
 45.000000|-8.000000
 45.000000|-30.000000
-CZQM|0|1|2|41.866667|-68.681703|48.500000|-56.051667|45.183334|-62.366685
+CZQM|0|1|4|41.866667|-68.681703|48.500000|-56.051667|45.183334|-62.366685

```

Comment:
This is just a line, is this intended?
![image](https://user-images.githubusercontent.com/42404718/99902491-84e5e780-2cbe-11eb-8f66-63f325f2ad8b.png)